core/src/main/scala/io/finch/endpoint/header.scala

```diff
@@ -1,39 +1,46 @@
 package io.finch.endpoint
 
+import scala.reflect.ClassTag
+
+import com.twitter.util.{Return, Throw}
 import io.finch._
 import io.finch.internal._
 import io.finch.items._
 
-private abstract class Header[A](name: String) extends Endpoint[A] {
+private abstract class Header[A](name: String)(implicit d: DecodeEntity[A], tag: ClassTag[A]) extends Endpoint[A] {
```

Comment:
Let's pass `DecodeEntity` and `ClassTag` as explicit parameters (as I did in #873). There is no value for them to be implicit.